packages/jupyter-ai/.eslintrc.js

```diff
@@ -27,13 +27,32 @@ module.exports = {
     '@typescript-eslint/no-explicit-any': 'off',
     '@typescript-eslint/no-namespace': 'off',
     '@typescript-eslint/no-use-before-define': 'off',
-    '@typescript-eslint/quotes': [
+    '@stylistic/quotes': [
```

Comment:
This is because `typescript-eslint` moved stylistic icons to stylistic, see https://typescript-eslint.io/rules/quotes/